src/commands/node/info.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import BaseIPCCommand from '../../base_ipc';
+
+export default class InfoCommand extends BaseIPCCommand {
+	static description = 'Gets node information';
+
+	static examples = ['node:info'];
+
+	async run(): Promise<void> {
+		try {
+			const nodeInfo = await this._channel.invoke<string>('app:getNodeInfo');
```

Comment:
```suggestion
			const nodeInfo = await this._channel.invoke<Record<string, unknown>>('app:getNodeInfo');
```
i think nodeInfo is not string?